src/Apps/Order/Routes/Details/Components/OrderDetails.tsx

```diff
@@ -0,0 +1,173 @@
+import ShieldIcon from "@artsy/icons/ShieldIcon"
+import {
+  Box,
+  Column,
+  Flex,
+  GridColumns,
+  Image,
+  Join,
+  Message,
+  ResponsiveBox,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { type Brand, BrandCreditCardIcon } from "Components/BrandCreditCardIcon"
+
+interface OrderDetailsProps {
+  orderId: string
+}
+
+export const OrderDetails = ({ orderId }: OrderDetailsProps) => {
+  // const orderData = useFragment(ORDER_FRAGMENT, order)
+
+  return (
+    <GridColumns>
+      <Column span={[12, 8]} start={[1, 3]}>
+        {/* <Box bg="red">Order ID: {orderId}</Box> */}
+
+        <Spacer y={2} />
+        {/* Title */}
+        <Text variant="lg">Great choice, Viviana! </Text>
+        {/* Order # */}
+        <Text variant="xs">Order #123456789 </Text>
+
+        <Spacer y={4} />
+        {/* Message */}
+        <Join separator={<Spacer y={1} />}>
```

Comment:
This works fine, but can also use the `Stack` component with the gap prop (ideal), or `Flex flexDirection="column" gap={1}`